amundsen_application/static/css/styles.scss

```diff
@@ -16,3 +16,8 @@ form {
   margin-bottom: 0;
 }
 
+.container {
+  &.amundsen-crumb {
```

Comment:
This should go into a new component.

I think every page should have the standard top-margin (64px) whether or not a crumb is present. The crumb itself is exempt from the standard margin and can do so with absolute positioning or a negative margin-top (personal preference). There are a few options here.